_includes/releases/v23.1/v23.1.0-alpha.4.md

```diff
@@ -25,7 +25,7 @@ Release Date: February 27, 2023
 
 <h3 id="v23-1-0-alpha-4-operational-changes">Operational changes</h3>
 
-- Added COCKROACH_RAFT_LOG_QUEUE_CONCURRENCY env var which controls the number of parallel workers doing Raft log truncations. It can be used to make the in-memory log truncations more agressive and reduce the amount of Raft log data flushed to disk. [#97029][#97029]
+- Add COCKROACH_RAFT_LOG_QUEUE_CONCURRENCY env var which controls the number of parallel workers doing Raft log truncations. It can be used to make the in-memory log truncations more agressive and reduce the amount of Raft log data flushed to disk. [#97029][#97029]
```

Comment:
I think I had a suggested edit here but it is now hidden if so. The environment variable should be in code font, and "env var" should be changed to "environment variable".